tools/test/detect_targets_test.py

```diff
@@ -16,14 +16,59 @@
 """
 
 import unittest
+from mock import patch
 from tools.detect_targets import get_interface_version
-from tools.test_api import get_autodetected_MUTS_list
+
+
+class MbedLsToolsMock():
+    
+    def __init__(self, type):
+        self.interface_test_type = type
```

Comment:
Could you change the name of this parameter to not shadow a builtin?

Reply:
Done  👍

Reply:
💯